lms/static/js/verify_student/views/image_input_view.js

```diff
@@ -0,0 +1,114 @@
+/**
+ * Allow users to upload an image using a file input.
+ *
+ * This uses HTML Media Capture so that iOS will
+ * allow users to use their camera instead of choosing
+ * a file.
+ */
+
+ var edx = edx || {};
+
+ (function( $, _, Backbone, gettext ) {
+    'use strict';
+
+    edx.verify_student = edx.verify_student || {};
+
+    edx.verify_student.ImageInputView = Backbone.View.extend({
+
+        template: "#image_input-tpl",
+
+        initialize: function( obj ) {
+            this.$submitButton = obj.submitButton || "";
+            this.modelAttribute = obj.modelAttribute || "";
+            this.errorModel = obj.errorModel || null;
+        },
+
+        render: function() {
+            var renderedHtml = _.template( $( this.template ).html(), {} );
+            $( this.el ).html( renderedHtml );
```

Comment:
This is going to end up being difficult in our current test infrastructure, since the template isn't necessarily created when we execute these top-level statements.  This would be much easier with RequireJS and the text plugin, since we can guarantee that the template text is available before any of these views are created.